apps/web/tsconfig.json

```diff
@@ -0,0 +1,30 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```

Comment:
ehpr approach is dry, we can follow that for our project as well. they have a root-level tsconfig.json